buildSrc/src/main/kotlin/com/datadog/gradle/config/KotlinConfig.kt

```diff
@@ -17,4 +19,20 @@ fun Project.kotlinConfig() {
             jvmTarget = JavaVersion.VERSION_1_8.toString()
         }
     }
+
+    val moduleName = this@kotlinConfig.name
+    val javaAgentJar = File(File(rootDir, "libs"), "dd-java-agent-0.67.0.jar")
+    afterEvaluate {
```

Comment:
the way I see it the `taskConfig` already applies the code block inside the `afterEvaluate`